app/views/public/event_detail.py

```diff
@@ -141,8 +141,9 @@ def display_event_schedule(identifier):
         abort(404)
     tracks = DataGetter.get_tracks(event.id)
     accepted_sessions_count = get_count(DataGetter.get_sessions(event.id))
+    confirmed_sessions_count = get_count(DataGetter.get_sessions(event.id, state='confirmed'))
```

Comment:
@niranjan94 
What do you suggest ?
Should I make only one query getting all the sessions and then traversing till I get the first confirmed or accepted state session , then break the loop and set `accepted_sessions_count = 1` .
Or something else ?